content/docs/command-reference/plots/index.md

```diff
@@ -260,3 +277,48 @@ file:///Users/usr/src/plots/classes.csv.html
 
 > A confusion matrix [template](/doc/command-reference/plots#plot-templates) is
 > predefined in DVC (found in `.dvc/plots/confusion.json`).
+
+### Example for the HTML template
+
+Download the Vega libraries into the directory where you'll produce the
```

Comment:
Should the title be more specific? Something about **offline environments** or **without Internet access**.

Reply:
(And let's start with a quick motivation intro as mentioned earlier 🙂)